python/examples/advanced/String_Tracking.ipynb

```diff
@@ -0,0 +1,740 @@
+       "column                                                         \n",
+       "onlyDigits                                               2.0   \n",
+       "mixed                                                   23.0   \n",
+       "onlyAlpha                                                5.0   \n",
+       "\n",
+       "            unicode_range/unicode_range/string_length/q_75  \\\n",
+       "column                                                       \n",
+       "onlyDigits                                             2.0   \n",
+       "mixed                                                 31.0   \n",
+       "onlyAlpha                                              5.0   \n",
+       "\n",
+       "            unicode_range/unicode_range/string_length/q_90  \\\n",
+       "column                                                       \n",
+       "onlyDigits                                             3.0   \n",
+       "mixed                                                 32.0   \n",
+       "onlyAlpha                                              7.0   \n",
+       "\n",
+       "            unicode_range/unicode_range/string_length/q_95  \\\n",
+       "column                                                       \n",
+       "onlyDigits                                             3.0   \n",
+       "mixed                                                 32.0   \n",
+       "onlyAlpha                                              7.0   \n",
+       "\n",
+       "            unicode_range/unicode_range/string_length/q_99                type  \n",
+       "column                                                                          \n",
+       "onlyDigits                                             3.0  SummaryType.COLUMN  \n",
+       "mixed                                                 32.0  SummaryType.COLUMN  \n",
+       "onlyAlpha                                              7.0  SummaryType.COLUMN  \n",
+       "\n",
+       "[3 rows x 57 columns]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "profile_view_df = prof.view().to_pandas()\n",
+    "profile_view_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see there's a lot of different metrics for each of the original dataframe's columns. In the `unicode_range` namespace, we'll have additional sub namespaces. In this case, we have metrics for:\n",
+    "\n",
+    "- digits: distribution metrics for characters inside the unicode's digit range\n",
+    "- alpha: distribution metrics for characters inside the unicode's lowercase letters range\n",
+    "- UNKNOWN: distribution metrics for character that fall anywhere outside the predefined range (digits and alpha)\n",
+    "- string_length: distribution metrics for overall string length\n",
+    "\n",
+    "For each of these namespaces, we have metrics such as:\n",
```

Comment:
Is the correct vocabulary "metrics" (maybe sub metrics?) or "components" or ?

Reply:
Maybe:
- `unicode_range` is a metric
- `digits/alpha/etc` is a submetrics
- `mean/stddev/etc` is a metric component
?

Reply:
Solved by using the above terminology